public/chassis.css

```diff
@@ -0,0 +1,426 @@
+/* 
+ Table of contents:
+1. Global styles
+2. Utility classes
+3. Components 
+   3.1 Avatar
+   3.2 Badge
+   3.3 Card
+*/
+
+/* 
+ // Global styles 
+*/
+
+* {
+  font-family: "Open Sans", sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --color-1: #1f1f27;
+  --color-2: #f3f4f6;
+  --color-3: #dc2626;
+  --color-4: red;
+  --color-5: green;
+  --color-6: grey;
+  --color-7: #d1d5db;
+}
+
+button {
+  cursor: pointer;
+}
+
+/*
+ // Utility classes
+*/
+
+.border-round {
+  border-radius: 50%;
+}
+
+.icon {
+  width: 32px;
+  height: 32px;
+}
+
+.icon > svg {
+  width: 100%;
+  height: auto;
+}
+
+.font-sm {
+  font-size: 0.6em;
+}
+
+.font-md {
+  font-size: 0.9em;
+}
+

```

Comment:
You have used em I think You should use rem instead of em.

Reply:
Done. Changed complete CSS to rem unit-based.